services/ymax-planner/test/config.test.ts

```diff
@@ -203,6 +204,43 @@ test('loadConfig rejects empty required values', async t => {
   });
 });
 
+test('loadConfig accepts ymax0 contract instance', async t => {
+  const env = {
+    MNEMONIC: 'test mnemonic phrase',
+    ALCHEMY_API_KEY: 'test1234',
+    CONTRACT_INSTANCE: 'ymax0',
+  };
+  const secretManager = makeFakeSecretManager();
+
+  const config = await loadConfig(env, secretManager);
```

Comment:
It's probably a good time to reduce this boilerplate:
```suggestion
  const config = await callLoadConfig({ CONTRACT_INSTANCE: 'ymax0' });
```
with
```ts
/** Acceptable values for all required environment variables. */
const baseEnv = {
  MNEMONIC: 'test mnemonic phrase',
  ALCHEMY_API_KEY: 'test1234',
  CONTRACT_INSTANCE: 'ymax1',
};

const callLoadConfig = (
  envOverrides: Record<string, string | undefined> = {},
  secretManager: SecretManager = makeFakeSecretManager(),
) => loadConfig({ ...baseEnv, envOverrides }, secretManager);
```

Reply:
Done in #12063.